lapis2/src/main/kotlin/org/genspectrum/lapis/model/ModelAggregated.kt

```diff
@@ -0,0 +1,39 @@
+package org.genspectrum.lapis.model
+
+import org.genspectrum.lapis.response.AggregatedResponse
+import org.genspectrum.lapis.silo.And
+import org.genspectrum.lapis.silo.PangoLineageEquals
+import org.genspectrum.lapis.silo.SiloAction
+import org.genspectrum.lapis.silo.SiloClient
+import org.genspectrum.lapis.silo.SiloFilterExpression
+import org.genspectrum.lapis.silo.SiloQuery
+import org.genspectrum.lapis.silo.StringEquals
+import org.genspectrum.lapis.silo.True
+import org.springframework.stereotype.Component
+
+@Component
+class ModelAggregated(private val siloClient: SiloClient) {
+    fun handleRequest(filterParameter: Map<String, String>): AggregatedResponse {
+        if (filterParameter.isEmpty()) {
+            return siloClient.sendQuery(
+                SiloQuery(SiloAction.aggregated(), True),
+            )
+        }
+
+        return siloClient.sendQuery(
+            SiloQuery(
+                SiloAction.aggregated(),
+                And(
+                    filterParameter.map { convertToSiloFilter(it.key, it.value) },
+                ),
+            ),
+        )
+    }
+
+    fun convertToSiloFilter(key: String, value: String): SiloFilterExpression {
+        if (key == "pango_lineage") {
```

Comment:
In the old LAPIS, it's `pangoLineage`. I think I'd prefer that, since it's more standard to use camelCase in HTTP and JSON (at least that's my impression).